quickstarts/consolequickstart-install-cryostat.yaml

```
@@ -0,0 +1,144 @@
+
+    1. You can create JVM targets that correspond to the JVM instances that you use to run your containerized workload. 
+
+    1. You can connect Cryostat to the JVM targets to record and analyze data about heap and non-heap memory usage, thread count, garbage collection, and other performance metrics for each JVM target.
+
+    To start using Cryostat, install the cert-manager Operator for Red Hat OpenShift and Red Hat build of Cryostat Operators on your cluster.
+
+  tasks:
+    - title: Install the cert-manager Operator for Red Hat OpenShift
+      description: >
+        To install the **cert-manager Operator for Red Hat OpenShift**:
+
+        1. From the **Administrator** perspective, go to the **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. In the **Filter by keyword** field, type `cert-manager`.
+          
+          **Note:** If the tile has an **Installed** label, the Operator is already installed. Proceed to the next task to install the **Red Hat build of Cryostat** Operator.
+        
+        1. Click the **cert-manager Operator for Red Hat OpenShift** tile to open the Operator details.
+        
+        1. In the **cert-manager Operator for Red Hat** OpenShift panel click **Install**.
+        
+        1. The **Install Operator** page opens with default values.
+        
+        1. Select the default channel `stable`.
+        
+        1. Click **Install**. A page opens that shows the various installation steps and displays a message when the Operator is installed.
+        
+        1. Click **View installed Operators in Namespace** to open the Installed Operators page.
+      review:
+        failedTaskHelp: >-
+          This task is incomplete. Try the task again, or [read
+          more](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-operator-install)
+          about this topic.
+        instructions: >-
+          #### Verify that **cert-manager Operator for Red Hat OpenShift** is installed:
+
+            - From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+            - The cert-manager Operator for Red Hat OpenShift should be shown in the list of Operators
+            - In the **Status** column, do you see the status of **cert-manager Operator for Red Hat OpenShift** as **Succeeded**?
+
+    - title: Install the Red Hat build of Cryostat Operator
+      description: >
+        To install the Red Hat build of Cryostat operator:
+       
+        1. From the **Administrator** perspective, go to the **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. In the **Filter by keyword** field, type `Red Hat build of Cryostat`.
```

Working through these steps, I noticed I ended up with a Cryostat CR in the `openshift-operators` namespace, which is likely not where the user's Java workloads are. I think we need an additional step here to tell the user to change to the OpenShift project containing their workloads.

```suggestion
        1. From the **Project** list, select the OpenShift project that hosts your containerized Java applications.
        
        1. In the **Filter by keyword** field, type `Red Hat build of Cryostat`.
```